src/compiler/diagnosticMessages.json

```diff
@@ -1051,6 +1051,10 @@
         "category": "Error",
         "code": 1360
     },
+    "A bigint literal cannot use exponential notation lower than ES2016.": {

```

Comment:
this error is not limited to literals. consider removing "literal" from the message

Reply:
Can you move this lower down where other semantic errors are? Also, consider

`Exponentiation cannot be performed on 'bigint' values unless the 'target' option is set to 'es2016' or later.`